addons/godot-firebase/firebase/firebase.gd

```diff
@@ -27,23 +46,23 @@ var config : Dictionary = {
     "appId": "",
     "clientId": "",
     "clientSecret": "",
-    }
+    "domainUriPrefix": "",
+}
 
-func load_config() -> void:
-    if ProjectSettings.has_setting(ENVIRONMENT_VARIABLES+"apiKey"):
-        for key in config.keys():
-            if ProjectSettings.get_setting(ENVIRONMENT_VARIABLES+key)!="":
-                config[key] = ProjectSettings.get_setting(ENVIRONMENT_VARIABLES+key)
+func _load_config() -> void:
+    if ProjectSettings.has_setting(_ENVIRONMENT_VARIABLES+"apiKey"):
+        for key in _config.keys():
+            if ProjectSettings.get_setting(_ENVIRONMENT_VARIABLES+key)!="":
+                _config[key] = ProjectSettings.get_setting(_ENVIRONMENT_VARIABLES+key)
     else:
         print("No configuration settings found, add them in override.cfg file.")
 
 func _ready() -> void:
-    load_config()
-    Auth.set_config(config)
-    Firestore.set_config(config)
-    Database.set_config(config)
-    Storage.set_config(config)
-    DynamicLinks.set_config(config)
+    _load_config()
+    Auth._set_config(_config)
+    Firestore._set_config(_config)
+    Database._set_config(_config)
+    Storage._set_config(_config)
```

Comment:
Apparently this is missing the DynamicLinks._set_config(_config).

Reply:
Fixed!